src/SpikeCore/SpikeCore.Web/Startup.cs

```diff
@@ -171,13 +176,22 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
                 app.UseEndpoints(endpoints =>
                 {
-                    endpoints.MapHub<TestHub>("/hubs/test");
+                    endpoints.MapHub<BotConsoleHub>("/hubs/bot-console");
                     endpoints.MapControllerRoute(
                         name: "default",
                         pattern: "{controller=Home}/{action=Index}/{id?}");
                     endpoints.MapRazorPages();
+
+                    // Disable registration - user management only happens via the bot.
+                    endpoints.MapGet("/Identity/Account/Register",
+                        context => Task.Factory.StartNew(() =>
```

Comment:
Why? Does this require a `Task` result? `Task.FromResult` to return a sync result as a `Task`, or `Task.CompletedTask` for the `void` case. Troubling the threadpool just because we need a `Task` is really over the top :trollface:

Reply:
Wound up with this instead:

```
endpoints.MapGet("/Identity/Account/Register",
    context =>
        {
            context.Response.Redirect("/Identity/Account/AccessDenied", true, true);
            return Task.CompletedTask;
        }
);
```

Because `Redirect` is a void.